rosbag2_py/src/rosbag2_py/_reader.cpp

```diff
@@ -12,7 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Ignore -Wunused-value for clang.
+// Based on https://github.com/pybind/pybind11/issues/2225
```

Comment:
could we consolidate this block into a single place, something like `rosbag2_py/pybind.hpp`?

Reply:
Done in https://github.com/ros2/rosbag2/pull/529/commits/b8438fbaf9e0626c60d6c1e9f9704726ea73f0fa.